frontend/src/global_styles/openproject/_mixins.sass

```diff
@@ -214,3 +214,11 @@ $scrollbar-size: 10px
       cursor: default
       border-color: #c7c7c7
       background: rgba(218,223,225,0.75)
+
+@mixin unset-button-styles
+  padding: 0
+  margin: 0
+  background: 0
+  border: 0
+  font: inherit
+  text-align: left
```

Comment:
I'd also inherit the text-align, and I'm not sure what to do with `outline`. We might want to wait for a designer to tell us how to handle hover/focus styles.